File: Components/Reporting/CS/ReportMainDemo/NorthwindTraders/CatalogPreviewControl.cs
using System;

namespace XtraReportsDemos.NorthwindTraders {
    public class CatalogPreviewControl : XtraReportsDemos.PreviewControl {
        protected override DevExpress.XtraReports.UI.XtraReport CreateReport() {
            return new CatalogReport();
        }
    }
}

File: Components/Reporting/VB/ReportMainDemo/NorthwindTraders/CatalogPreviewControl.vb
Imports System

Namespace XtraReportsDemos.NorthwindTraders
    Public Class CatalogPreviewControl
        Inherits XtraReportsDemos.PreviewControl

        Protected Overrides Function CreateReport() As DevExpress.XtraReports.UI.XtraReport
            Return New CatalogReport()
        End Function
    End Class
End Namespace
